firebase-vscode/src/data-connect/code-lens-provider.ts

```diff
@@ -72,84 +72,93 @@ export class OperationCodeLensProvider extends ComputedCodeLensProvider {
   ): vscode.CodeLens[] {
     // Wait for configs to be loaded and emulator to be running
     const fdcConfigs = this.watch(dataConnectConfigs)?.tryReadValue;
-    const projectId = this.watch(firebaseRC)?.tryReadValue?.projects.default;
     if (!fdcConfigs) {
       return [];
     }
-
-    const codeLenses: vscode.CodeLens[] = [];
+    const projectId = this.watch(firebaseRC)?.tryReadValue?.projects.default;
 
     const documentText = document.getText();
-    // TODO: replace w/ online-parser to work with malformed documents
```

Comment:
This TODO still makes sense. Mostly so that the codelens do not disappear and appear repeatedly while you're typing. Let's leave this here even if you don't want to address it any time soon

Reply:
The GQL language server we already depend on has an online parser, but we haven't got the time to pipe its output to our code lens provider just yet.